task01.rb

```diff
@@ -0,0 +1,10 @@
+#1. Write a script that returns true if a string contains comparison, e.g. "<23", ">45".
+
+def task01(str)
+  str.gsub!(/>>/, "")
```

Comment:
@MaraEfimova In case of `>>>1` the script would output `true`, which in fact is invalid input, in such case the script should throw an error (error type and message are up to you).